guides/v3.17.0/in-depth-topics/patterns-for-components.md

```diff
@@ -1,7 +1,6 @@
 This guide assumes that you have read the [component guides](../../components/)
 and are familiar with how Ember components work. We'll cover components in more
-depth in this section, including additional functionality that was not covered
-in the main guides, and patterns
+depth in this section, including additional functionality and patterns.
```

Comment:
The existing phrase "additional functionality" feels vague to me. I think it'd be also good to explicitly mention in the intro that component patterns will be presented at the end.

What do you think of:

```suggestion
depth in this section. At the end, we'll present recommended component patterns.
```